tests/unit/containers/my-cooperations/add-review-modal/AddReviewModal.spec.jsx

```diff
@@ -0,0 +1,136 @@
+import { describe, expect, vi } from 'vitest'
+import { render, screen } from '@testing-library/react'
+
+import { UserRoleEnum } from '~/types/user/user.index'
+import userEvent from '@testing-library/user-event'
+
+import AddReviewModal from '~/containers/my-cooperations/add-review-modal/AddReviewModal'
```

Comment:
Please, consider some imports order. Imports from libs should be on the top of the file

```suggestion
import { describe, expect, vi } from 'vitest'
import { render, screen } from '@testing-library/react'
import userEvent from '@testing-library/user-event'

import { UserRoleEnum } from '~/types/user/user.index'

import AddReviewModal from '~/containers/my-cooperations/add-review-modal/AddReviewModal'
```